doc/src/tools/kani.md

```diff
@@ -88,14 +88,14 @@ pub mod verify {
 
 Run the following command in your local terminal:
 
-`kani verify-std -Z unstable-options "path/to/library" --target-dir "/path/to/target" -Z function-contracts -Z stubbing`.
+`kani verify-std -Z unstable-options "path/to/library" --target-dir "/path/to/target" -Z function-contracts -Z stubbing -Z mem-predicates`.
```

Comment:
Yes, thank you, you are right. But then I do wonder whether `-Z stubbing` is even necessary.

Reply:
Removing `-Z stubbing` does not create an issue for the compilation process so I think it's safe to remove it.